.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- [ ] Great job on adding the linter configuration 👍 . But there are some tests that did not pass. Please have a look at the test details to know more about what parts of your code caused the issues, and fix it 👍 .